### ISSUE TITLE
signature verification failed!

### ISSUE THREAD

**Issue by vickencholakian** [micahflee/torbrowser-launcher#344]
I am getting this message...
![screenshot at 2018-09-07 06-52-56](https://user-images.githubusercontent.com/43065487/45215467-21994780-b263-11e8-8435-515df88d2e8d.png)

**Comment by WMDwebs** [comment]
same thing here after the browser update...

**Comment by rogers0** [comment]
@vickencholakian @WMDwebs if you use debian, try experimental version [1].

[1] https://wiki.debian.org/TorBrowser#Debian_.22experimental.22

It's working on my environment.

**Comment by hThoreau** [comment]
I'm getting the same error, and tried the recommendation by @rogers0. But it seems that the version of torbrowser-launcher I got from backports was already the most recent.
```
Reading package lists... Done
Building dependency tree
Reading state information... Done
torbrowser-launcher is already the newest version (0.2.9-1~bpo9+1).
0 upgraded, 0 newly installed, 0 to remove and 102 not upgraded.
> 
```
Another recently closed thread on this same topic (https://github.com/micahflee/torbrowser-launcher/issues/342#issuecomment-418981809) seems to suggest that tis problem can be soulved by refreashing the torbrowser-launcher GPG keys. Though I got an error for that as well:
```
> gpg --homedir "$HOME/.local/share/torbrowser/gnupg_homedir/" --refresh-keys --keyserver pgp.mit.edu
gpg: WARNING: unsafe permissions on homedir '/home/user/.local/share/torbrowser/gnupg_homedir'
gpg: connecting dirmngr at '/run/user/1000/gnupg/d.dniezwd1guds6p6c6uwjr6bf/S.dirmngr' failed: IPC connect call failed
gpg: keyserver refresh failed: No dirmngr
>      
```
Regards,

**Comment by rogers0** [comment]
@hThoreau 0.2.9-1~bpo9+1 is not latest.
since it seems you're using stretch, please refer:
- https://wiki.debian.org/TorBrowser#Debian_9_.22Stretch.22

> Though I got an error for that as well:

maybe you can also try to install a backported gnupg:

```
$ apt install -t stretch-backports gnupg
```

**Comment by palkinev** [comment]
I had exactly the same error as @vickencholakian.

I am also using debian stretch with stretch-backports, 
and I installed tor-browser by following page on debian website, which @rogers0 referred above.

And as @rogers0 suggested I installed gnupg from stretch-backports and it have solved this issue for me.

**Comment by micahflee** [comment]
This is fixed in the new version 0.3.0 that I just released. You can get it by either [installing from source](https://github.com/micahflee/torbrowser-launcher/blob/develop/BUILD.md) or installing from the Ubuntu PPA, described [here](https://github.com/micahflee/torbrowser-launcher/blob/develop/README.md#installing-in-ubuntu).
